### PR TITLE
Coffee news site -  my first project

### PR DESCRIPTION
**Pull request by lilihorvath**
I had some problem first the responsive and after 3days I figured out that in the meta tag one of the - character was an =... But I finished it!

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,120 @@
...
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" sizes="16x16" href="favicon.png">
```
Comment (HIPPIEKICK) [inline code/index.html:8]: ☕ 💯

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,120 @@
...
+</nav> 
+</header>
 
-    <section class="">
-      <!-- The other news  -->
+<!-- MAIN ARTICLE-->
+<section class="main-article">
+ <a href="https://www.nbcnews.com/health/health-news/coffee-good-you-more-science-shows-n888356" target="_blank">
+    <section class="container">
+        <img src="girl-coffee.jpg" alt="girl-drinks-coffee">
```
Comment (HIPPIEKICK) [inline code/index.html:33]: No need for dashes in the alt tag! And just a heads up, I think this image file is super big (it takes a few seconds to load fully).

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,120 @@
...
+            <h2>Coffee is good for you, more science shows</h2>
+            <p>People who drank coffee, no matter how much or what kind they drank, were less likely to die over a 10-year period than non-coffee drinkers, according to a new report.</p>
+        </div>
     </section>
-  </body>
-</html>
+ </a>
+</section>
+
+<a href="#top" class="myBtn" title="Go to top">⌃</a>
```
Comment (HIPPIEKICK) [inline code/index.html:42]: To avoid confusion, keep the abbreviations to a minimum

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,120 @@
...
+
+<section class="cards-container">
+  <section class="container">
```
Comment (HIPPIEKICK) [inline code/index.html:45]: I don't think you need two sections here

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,120 @@
...
+  <section class="icon-row">
+    <div class="social-menu">
+     <ul>
+         <li> <a href="#"><i class="fa fa-facebook"></i></a></li>
+         <li> <a href="#"><i class="fa fa-twitter"></i></a></li>
+         <li> <a href="#"><i class="fa fa-instagram"></i></a></li>
+         <li> <a href="#"><i class="fa fa-youtube"></i></a></li>
+     </ul>
+    </div>
+</section>
```
Comment (HIPPIEKICK) [inline code/index.html:112]: Nice! Just think about the indentation:
```
<section class="icon-row">
  <div class="social-menu">
    <ul>
      <li>
        <a href="#">
          <i class="fa fa-facebook"></i>
        </a>
      </li>
```
etc, to make it more readable. Every child should be indented one extra level

File: code/index.html [hunk 6]
```diff
@@ -1,22 +1,120 @@
...
+
+<!--FOOTER-->
+<footer>
+    <a href="#"><img class="footer-logo" src="logo.png" alt="logo" width="25px"> © Espresso Crema 2020 </a>
+</footer>
```
Comment (HIPPIEKICK) [inline code/index.html:117]: Don't forget to close the img-tag at the end! And try to keep all styling in the CSS-file

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,312 @@
...
+h4 {
+  font-size: 32px;
+  font-weight: 100;
+  margin-top: -16px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1400px;
+  margin: 0 auto;
```
Comment (HIPPIEKICK) [inline code/style.css:36]: These two make your page look extra nice when it comes to the card-section. Just a heads up - it's also cropping your background for the main-article

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,312 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div{
+  width: 30px;
+  height: 5px;
+  background: #515151;
+  border-radius: 5px;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:66]: Nice!

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,312 @@
...
+  }
+
+  .cards a {
+    width: 15%;
+    flex-grow:1;
+    margin: 16px;
+  }
+
+  .cards a:hover {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (HIPPIEKICK) [inline code/style.css:310]: This looks super nice!

**Review by HIPPIEKICK — APPROVED** [review]
Well done with the responsiveness!

The indentation seems to be varying between 4 and 2 spaces. Check your VSC settings so that the tab size is set to 2 spaces. You have a lot of small things that make your page look nice, the up-button, the box-shadow on the cards, the social media icons, the favicon and more.

Keep up the good work Lili!

File: code/index.html (hunk 2 above)
Comment (lilihorvath) [reply code/index.html:33]: Ojj, sorry! Next time I am going to check it !

File: code/index.html (hunk 4 above)
Comment (lilihorvath) [reply code/index.html:45]: I am going to correct it!

File: code/index.html (hunk 5 above)
Comment (lilihorvath) [reply code/index.html:112]: Ooo, fine! I'm going to do that next time, thank you!

File: code/index.html (hunk 6 above)
Comment (lilihorvath) [reply code/index.html:117]: Definitely, thank you!

File: code/style.css (hunk 7 above)
Comment (lilihorvath) [reply code/style.css:36]: Ooo, and how I can keep that just for the card section? Thank you for your answer!

Comment (HIPPIEKICK) [reply code/style.css:36]: One idea is to give the card section another class name to be able to have two lines of code that differs from the main container. Like:
```
.main-container, .container {
/*Styling applying to both*/
}

.container {
/*Styling applying for only the card section*/
}
```

**Pull request closed without merging**